task1/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+const createEl = React.createElement('h2', {className: 'create-element'}, 'Create element component!');
+
+const FuncComponent = (props) => (<h2>Create function component: "{props.message}"</h2>);
+
+class ClassComponent extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render () {
+    return (<h2>Create class component: "{this.props.message}"</h2>);
+  }
+}
+
+class ClassComponentPure extends React.PureComponent {
+  render () {
+    return (<h2>Create pure component: "{this.props.message}"</h2>);
```

Comment:
Just to let you know, if you return only one component you can do it without parenthesis in this example `return <h2>Create pure component: "{this.props.message}"</h2>;`